pkg/graph/schema.graphql

```diff
@@ -15,6 +15,7 @@ type AccessibilityRequest {
   documents: [AccessibilityRequestDocument!]!
   id: UUID!
   name: String!
+  relevantTestDate: TestDate
```

Comment:
Nice! I was thinking this might just be a date, but we might eventually want the score, too. 👍

Reply:
How do we define "relevant" here? I'm not sure how to interpret this. Either just `testDate` or something like `latestTestDate` might be clearer.

Reply:
Gah, sorry I messed up GH threading here. I responded to Jim's question about "relevant" at the top level.